test/source/mock/key-manager/key-manager-endpoints.ts

```diff
@@ -209,6 +206,20 @@ yeSm0uVPwODhwX7ezB9jW6uVt0R8S8iM3rQdEMsA/jDep5LNn47K6o8VrDt0zYo6
 
 export const MOCK_KM_LAST_INSERTED_KEY: { [acct: string]: { decryptedPrivateKey: string, publicKey: string } } = {}; // accessed from test runners
 
+export const LIVE_KM_RESPONSE: { privateKeys: { decryptedPrivateKey: string }[] } = { privateKeys: [] };
+
+export const liveKeyManagerEndpoints: HandlersDefinition = {
+  '/flowcrypt-email-key-manager/keys/private': async ({ }, req) => {
+    if (isGet(req)) {
+      return LIVE_KM_RESPONSE;
+    }
+    if (isPut(req)) {
+      throw new HttpClientErr(`Unexpectedly calling liveKeyManagerEndpoints:/keys/private PUT`);
+    }
+    throw new HttpClientErr(`Unknown method: ${req.method}`);
+  }
+};
```

Comment:
It won't have to be a separate definition. It could go below to one of the accounts, and then you can differentiate just by the account email (it's all mocked so you can give it any account email for login). It can still be returning `LIVE_KM_RESPONSE` as you've done it. That should simplify this surrounding code / minimize changes.

Reply:
In this case a live idToken will be submitted, so I'll have to create a mapping entry in `OauthMock.acctByIdToken` first, so a lot of code anyway. My initial concern was: do we really need to start the whole set of mocks for live Gmail tests (EKM, FES etc.) or just a smaller subset (EKM only)?

Reply:
Ah! You are writing it as a live gmail test. I didn't think about that / didn't realize that.

I wish it could be written as a completely mocked test, somehow. Because it has nothing to do with Gmail webpage in particular.

For example, you Gmail API mock could serve a mock Gmail page. That one would maybe contain a copy-pasted snapshot of Gmail page HTML, trimmed down. Enough to make the content scripts happy. Then the mocked tests would also get a slightly different manifest, to run their content scripts on this mock of a gmail page.

What do you think? It's more work now but I think it's more maintainable, and we could also move some live gmail tests over here, when they don't rely on Gmail webpage specifics too much (like tests where we are just checking for a setup notification on gmail page - that would be for later).